.github/CONTRIBUTING.md

```diff
@@ -95,3 +95,19 @@ If you are using a new feature from core Nextflow, you may bump the minimum requ
 ### Images and figures
 
 For overview images and other documents we follow the nf-core [style guidelines and examples](https://nf-co.re/developers/design_guidelines).
+
+## GitHub Codespaces
```

Comment:
What prompted this?

Reply:
see above...linting failed because nf-core made these changes to CONTRIBUTING.md, so I just scooped in the changes that now ship when anyone uses `nf-core create` so that linting passes